src/components/Search/CompareWithBase.tsx

```diff
@@ -302,19 +294,24 @@ function CompareWithBase({
             }
             formIsDisplayed={formIsDisplayed}
           />
+          {}
           <Grid
             item
             xs={2}
-            className={`${dropDownStyles.dropDown} ${bottomStyles.container}`}
+            display='flex'
+            justifyContent={hasEditButton ? 'flex-end' : 'space-between'}
+            className={`${dropDownStyles.dropDown}`}
+            alignItems='flex-end'
           >
-            <FrameworkDropdown
-              compact={false}
-              frameworkId={frameWorkId}
-              onChange={(event: SelectChangeEvent) => {
-                const id = +event.target.value as Framework['id'];
-                setframeWorkValue(id);
-              }}
-            />
+            {!hasEditButton && (
+              <FrameworkDropdown
+                frameworkId={frameWorkId}
```

Comment:
Don't know if it's a type or not. Would it be better to keep the `framework` all lowercase everywhere?

Reply:
Yes, I agree. I actually end up eliminating this state with the new 'framework' name in the next commit since it's not needed.